recipes-browser/webkitgtk/webkitgtk_2.32.1.bb

```diff
@@ -12,13 +12,13 @@ DEPENDS = "zlib libsoup-2.4 curl libxml2 cairo libxslt libidn \
            libwebp harfbuzz glib-2.0 gettext-native glib-2.0-native \
            sqlite3 libgcrypt"
 
-SRC_URI = "https://www.webkitgtk.org/releases/webkitgtk-${PV}.tar.xz \
-           file://musl.patch \
-           file://0001-ICU-69-deprecates-ubrk_safeClone-in-favor-of-ubrk_cl.patch \
-           file://0001-Move-cloneUBreakIterator-declaration-to-IntlWorkarou.patch \
-           file://0001-MiniBrowser-Fix-reproduciblity.patch \
+SRC_URI = "https://www.webkitgtk.org/releases/webkitgtk-${PV}.tar.xz;name=tarball \
+           file://musl.patch;name=musl \
```

Comment:
The musl patch should be removed.
It is now merged upstream in https://trac.webkit.org/changeset/275670 that was later corrected on https://trac.webkit.org/changeset/276695
Both changes are merged on 2.32.1

Reply:
I've  noticed some changes are already on upstream but others still not: https://github.com/Igalia/meta-webkit/blob/40cb794e5b542ae6d7a8c070d310d7775b6b9638/recipes-browser/webkitgtk/webkitgtk/musl.patch 

What I did is to update the musl.patch by removing the changes already applied but leaving there those what are still not in the tarball release.

Reply:
Those changes were made obsolete by https://trac.webkit.org/changeset/276695 which is merged on 2.32.1

Reply:
That's right, a build of 2.32.1 directly from a tarball builds and works fine with Musl without needing additional patching; I also removed all the Musl-specific patching when [updating the packaging in Buildroot](https://patchwork.ozlabs.org/project/buildroot/patch/20210518151407.264769-4-aperez@igalia.com/).